### ISSUE TITLE
3.4 — Baseline Result Summary

### ISSUE THREAD

**Issue by A01799073** [A01799073/AguacAIte-Labs#13]
Consolidate baseline results into a comparison table and write key observations.
Detail : 
The summary should (ipynb):

- Compare KNN, Decision Tree, and SVM

- Include key metrics (accuracy, precision, recall)

-Highlight strengths and limitations of each model

**This will serve as a reference point before introducing deep learning models.**

**Comment by A01799073** [comment]
**Review:**  
- KNN provides a reasonable baseline performance, but shows sensitivity to local noise when using small datasets (Our current basedata is short, it has 10 normal lung images and 10 lungs with tuberculosis images).

- KNN provides a reasonable baseline performance, but shows sensitivity to local noise when using small datasets.
- Decision Trees achieve perfect performance on the training/test split, indicating strong fitting capacity but a high risk of overfitting in small datasets.

**Summary**
KNN achieved moderate accuracy (83%),
Both Decision Tree and SVM models reached perfect accuracy, precision, and recall, suggesting strong separability of the classes under limited data conditions.
However, such results are indicative of potential overfitting and highlight the limitations of classical models in terms of generalization, motivating the use of convolutional neural networks in subsequent experiments.
